README.md

```diff
@@ -29,6 +29,7 @@ This client supports the following Google Cloud Platform services at a [Beta](#v
 * [Cloud Pub/Sub](#cloud-pubsub-beta) (Beta)
 * [Stackdriver Monitoring API](#stackdriver-monitoring-api-beta) (Beta)
 * [Stackdriver Trace](#stackdriver-trace-beta) (Beta)
+* [Cloud Dataproc](#cloud-dataproc) (Beta)
```

Comment:
It is generally not a good idea to add these before the gem is released. There may be several days between when this PR is merged and a release happens, and users may be confused why they can't download the gem. I would prefer to see changes to this file removed.

Reply:
Removed.